src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -0,0 +1,130 @@
+// Data attributed to https://iextrading.com/api-exhibit-a
+
+const ApiConsumer = require('../utils/api-consumer');
+
+
+let StockConsumer = new ApiConsumer('iex-trading', 'https://api.iextrading.com/1.0',{cache: {ttl: 3600*24*30*6}});
+
+const queryOpts = {
+    queryString: '/stock/',
+    headers:{
+    },
+    body: undefined,
+    cache: true
+};
+
+let stockParser = resp => {
+    let results = [];
+    results = resp[company];
+	    
+    return results;
+};
+
+/**
+ * Get the selling price of the specified company, with a 15 min delay
+ * @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ * @returns {double} Value of the current price for that stock
```

Comment:
`double` is not technically a JS type (nor an additional type, such as Latitude, Longitude, introduced in NetsBlox). Could you replace it with "Number"?

Also, minor thing but it might read better as "Current price of the given stock"